include/alpaka/exec/ExecGpuCudaRt.hpp

```diff
@@ -496,7 +496,7 @@ namespace alpaka
                             exec::cuda::detail::cudaKernel<TDim, TIdx, TKernelFnObj, TArgs...><<<
                                 gridDim,
                                 blockDim,
-                                blockSharedMemDynSizeBytes,
```

Comment:
Is this a bugfix? If so we should move it to a separate PR because this change not fits to the description.

Reply:
This fixes a warning that only occurs when ALPAKA_DEBUG==2 and was an error with Werror. I could create a new PR with the fix if you want.

Reply:
That would be nice because this is much easier to back-port.

Reply:
The fix has been merged separately